mkdocs.yml

```diff
@@ -35,9 +35,52 @@ theme:
         name: Switch to light mode
 
 nav:
-  - home: index.md
-  - debugging.md
-  - migration.md
+  - index.md
+  - Installation:
+      - Installation: installation/index.md
+      - installation/basic-install.md
+      - installation/creating-a-minikube-cluster-for-testing.md
+      - installation/helm-install-on-existing-cluster.md
+  - User Guide:
+      - user-guide/admin-user-account-configuration.md
+      - user-guide/network-and-tls-configuration.md
+      - user-guide/database-configuration.md
+  - Upgrade:
+      - upgrade/upgrading.md
+  - Uninstall:
+      - uninstall/uninstall.md
+  - Advanced Configuration:
```

Comment:
I should have looked at your PR first.  I organized these guides in my PR in order based on importance of a section for a new user (based on Matrix and mailing list history).  

I can re-order these as needed in a follow-up PR if you want.